### PR TITLE
Text Fixes

### PR DESCRIPTION
**Pull request by SgtBears**
Updates a few Noturn UHV texts.
Removes a few incorrect translations.
Improves the religious building's strategies and pedias.
Moved all Shrine's texts to the building texts file.
Fixes TXT_KEY_CIV_POLAND_EMPIRE.
Fixes TXT_KEY_BUILDING_MOAI_STATUES_HELP which was around the wrong way and didn't have COLOR_WARNING_TEXT.

### CODE REVIEW

File: Assets/XML/Text/Buildings.xml [hunk 1]
```diff
@@ -782,7 +774,7 @@
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_BUILDING_JEWISH_TEMPLE_STRATEGY</Tag>
-		<English>[COLOR_BUILDING_TEXT]Synagogues[COLOR_REVERT] ([COLOR_BUILDING_TEXT]Jewish Temples[COLOR_REVERT]) increase a city's happiness and culture.</English>
```
Comment (dguenms) [inline Assets/XML/Text/Buildings.xml:785]: Why singular?

File: Assets/XML/Text/Buildings.xml [hunk 2]
```diff
@@ -1023,7 +1067,7 @@
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_BUILDING_TAOIST_TEMPLE_STRATEGY</Tag>
-		<English>[COLOR_BUILDING_TEXT]Guans[COLOR_REVERT] ([COLOR_BUILDING_TEXT]Taoist Temples[COLOR_REVERT]) increase a city's happiness and culture.</English>
+		<English>The [COLOR_BUILDING_TEXT]Guan[COLOR_REVERT] ([COLOR_BUILDING_TEXT]Taoist Temples[COLOR_REVERT]) increases a city's happiness and culture. The [COLOR_BUILDING_TEXT]Guan[COLOR_REVERT] is required to build the [COLOR_BUILDING_TEXT]Taoist Gong[COLOR_REVERT] Building.</English>
```
Comment (dguenms) [inline Assets/XML/Text/Buildings.xml]: Temples is still plural

File: Assets/XML/Text/DynamicNames/DynamicNames_Poland.xml [hunk 3]
```diff
@@ -38,6 +38,10 @@
 		<English>Polish-Lithuanian Commonwealth</English>
 	</TEXT>
 	<TEXT>
+		<Tag>TXT_KEY_CIV_POLAND_EMPIRE</Tag>
```
Comment (dguenms) [inline Assets/XML/Text/DynamicNames/DynamicNames_Poland.xml]: This has already been fixed, and is probably the reason why the branch is incompatible. Please check for compatibility first.

File: Assets/XML/Text/Units.xml [hunk 4]
```diff
@@ -1331,7 +1331,7 @@
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_UNIT_JEWISH_MISSIONARY_STRATEGY</Tag>
-		<English>[COLOR_UNIT_TEXT]Rabbis[COLOR_REVERT] ([COLOR_UNIT_TEXT]Jewish Missionaries[COLOR_REVERT]) can spread [COLOR_HIGHLIGHT_TEXT]Judaism[COLOR_REVERT] in friendly cities (or rival cities with whom you have [COLOR_HIGHLIGHT_TEXT]Open Borders[COLOR_REVERT]).</English>
+		<English>The [COLOR_UNIT_TEXT]Rabbi[COLOR_REVERT] ([COLOR_UNIT_TEXT]Jewish Missionary[COLOR_REVERT]) can spread [COLOR_HIGHLIGHT_TEXT]Judaism[COLOR_REVERT] in the Middle East, Europe, Russia and the United States.</English>
```
Comment (dguenms) [inline Assets/XML/Text/Units.xml]: Same here, why singular?

**Comment by SgtBears** [comment]
Fixed the Polish_Empire.
Reverted the plurals, I had a reason for it but on a second examination I realised it wasn't a good reason.

File: Assets/XML/Text/Buildings.xml [hunk 5]
```diff
@@ -782,7 +774,7 @@
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_BUILDING_JEWISH_TEMPLE_STRATEGY</Tag>
-		<English>[COLOR_BUILDING_TEXT]Synagogues[COLOR_REVERT] ([COLOR_BUILDING_TEXT]Jewish Temples[COLOR_REVERT]) increase a city's happiness and culture.</English>
+		<English>[COLOR_BUILDING_TEXT]Synagogues[COLOR_REVERT] ([COLOR_BUILDING_TEXT]Jewish Temples[COLOR_REVERT]) increase a city's happiness and culture. [COLOR_BUILDING_TEXT]Synagogues[COLOR_REVERT] are required to build the [COLOR_BUILDING_TEXT]Jewish Sanhedrin[COLOR_REVERT] Building.</English>
```
Comment (dguenms) [inline Assets/XML/Text/Buildings.xml]: "building" should be lowercase here.

File: Assets/XML/Text/Buildings.xml [hunk 6]
```diff
@@ -814,7 +806,7 @@
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_BUILDING_ORTHODOX_TEMPLE_STRATEGY</Tag>
-		<English>[COLOR_BUILDING_TEXT]Churches[COLOR_REVERT] ([COLOR_BUILDING_TEXT]Orthodox Temples[COLOR_REVERT]) increase a city's happiness and culture.</English>
+		<English>[COLOR_BUILDING_TEXT]Churches[COLOR_REVERT] ([COLOR_BUILDING_TEXT]Orthodox Temples[COLOR_REVERT]) increase a city's happiness and culture. [COLOR_BUILDING_TEXT]Churches[COLOR_REVERT] are required to build the [COLOR_BUILDING_TEXT]Orthodox Cathedral[COLOR_REVERT] Building.</English>
```
Comment (dguenms) [inline Assets/XML/Text/Buildings.xml]: see above

File: Assets/XML/Text/Buildings.xml [hunk 7]
```diff
@@ -841,7 +841,7 @@
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_BUILDING_CATHOLIC_TEMPLE_STRATEGY</Tag>
-		<English>[COLOR_BUILDING_TEXT]Churches[COLOR_REVERT] ([COLOR_BUILDING_TEXT]Catholic Temples[COLOR_REVERT]) increase a city's happiness and culture. They also allows you to hire one [COLOR_HIGHLIGHT_TEXT]Priest[COLOR_REVERT] specialist.</English>
+		<English>[COLOR_BUILDING_TEXT]Churches[COLOR_REVERT] ([COLOR_BUILDING_TEXT]Catholic Temples[COLOR_REVERT]) increase a city's happiness and culture. [COLOR_BUILDING_TEXT]Churches[COLOR_REVERT] are required to build the [COLOR_BUILDING_TEXT]Catholic Cathedral[COLOR_REVERT] Building.</English>
```
Comment (dguenms) [inline Assets/XML/Text/Buildings.xml]: see above

File: Assets/XML/Text/Buildings.xml [hunk 8]
```diff
@@ -895,7 +931,7 @@
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_BUILDING_ISLAMIC_TEMPLE_STRATEGY</Tag>
-		<English>[COLOR_BUILDING_TEXT]Mosques[COLOR_REVERT] ([COLOR_BUILDING_TEXT]Islamic Temples[COLOR_REVERT]) increase a city's happiness and culture.</English>
+		<English>[COLOR_BUILDING_TEXT]Mosques[COLOR_REVERT] ([COLOR_BUILDING_TEXT]Islamic Temples[COLOR_REVERT]) increase a city's happiness and culture. [COLOR_BUILDING_TEXT]Mosques[COLOR_REVERT] are required to build the [COLOR_BUILDING_TEXT]Islamic Grand Mosque[COLOR_REVERT] Building.</English>
```
Comment (dguenms) [inline Assets/XML/Text/Buildings.xml]: see above

File: Assets/XML/Text/Buildings.xml [hunk 9]
```diff
@@ -916,13 +952,17 @@
...
+	</TEXT>
+	<TEXT>
 		<Tag>TXT_KEY_BUILDING_HINDU_TEMPLE</Tag>
 		<English>Hindu Mandir</English>
 		<German>Hinduistischer Mandir</German>
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_BUILDING_HINDU_TEMPLE_STRATEGY</Tag>
-		<English>[COLOR_BUILDING_TEXT]Mandirs[COLOR_REVERT] ([COLOR_BUILDING_TEXT]Hindu Temples[COLOR_REVERT]) increase a city's happiness and culture. The first player to build a [COLOR_BUILDING_TEXT]Mandir[COLOR_REVERT] founds Buddhism.</English>
+		<English>[COLOR_BUILDING_TEXT]Mandirs[COLOR_REVERT] ([COLOR_BUILDING_TEXT]Hindu Temples[COLOR_REVERT]) increase a city's happiness and culture. The first player to build a [COLOR_BUILDING_TEXT]Mandir[COLOR_REVERT] founds Buddhism. [COLOR_BUILDING_TEXT]Mandirs[COLOR_REVERT] are required to build the [COLOR_BUILDING_TEXT]Hindu Kshetra[COLOR_REVERT] Building.</English>
```
Comment (dguenms) [inline Assets/XML/Text/Buildings.xml]: see above

File: Assets/XML/Text/Buildings.xml [hunk 10]
```diff
@@ -959,7 +999,7 @@
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_BUILDING_BUDDHIST_TEMPLE_STRATEGY</Tag>
-		<English>[COLOR_BUILDING_TEXT]Stupas[COLOR_REVERT] ([COLOR_BUILDING_TEXT]Buddhist Temples[COLOR_REVERT]) increase a city's happiness and culture.</English>
+		<English>[COLOR_BUILDING_TEXT]Stupas[COLOR_REVERT] ([COLOR_BUILDING_TEXT]Buddhist Temples[COLOR_REVERT]) increase a city's happiness and culture. [COLOR_BUILDING_TEXT]Stupas[COLOR_REVERT] are required to build the [COLOR_BUILDING_TEXT]Buddhist Chaitya[COLOR_REVERT] Building.</English>
```
Comment (dguenms) [inline Assets/XML/Text/Buildings.xml]: see above

File: Assets/XML/Text/Buildings.xml [hunk 11]
```diff
@@ -980,13 +1020,17 @@
...
+	</TEXT>
+	<TEXT>
 		<Tag>TXT_KEY_BUILDING_CONFUCIAN_TEMPLE</Tag>
 		<English>Confucian Miao</English>
 		<German>Konfuzianischer Miao</German>
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_BUILDING_CONFUCIAN_TEMPLE_STRATEGY</Tag>
-		<English>[COLOR_BUILDING_TEXT]Miaos[COLOR_REVERT] ([COLOR_BUILDING_TEXT]Confucian Temples[COLOR_REVERT]) increase a city's happiness and culture.</English>
+		<English>[COLOR_BUILDING_TEXT]Miaos[COLOR_REVERT] ([COLOR_BUILDING_TEXT]Confucian Temples[COLOR_REVERT]) increase a city's happiness and culture. [COLOR_BUILDING_TEXT]Miaos[COLOR_REVERT] are required to build the [COLOR_BUILDING_TEXT]Confucian Xue Gong[COLOR_REVERT] Building.</English>
```
Comment (dguenms) [inline Assets/XML/Text/Buildings.xml]: see above

File: Assets/XML/Text/Buildings.xml [hunk 12]
```diff
@@ -1023,7 +1067,7 @@
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_BUILDING_TAOIST_TEMPLE_STRATEGY</Tag>
-		<English>[COLOR_BUILDING_TEXT]Guans[COLOR_REVERT] ([COLOR_BUILDING_TEXT]Taoist Temples[COLOR_REVERT]) increase a city's happiness and culture.</English>
+		<English>[COLOR_BUILDING_TEXT]Guans[COLOR_REVERT] ([COLOR_BUILDING_TEXT]Taoist Temples[COLOR_REVERT]) increase a city's happiness and culture. [COLOR_BUILDING_TEXT]Guans[COLOR_REVERT] are required to build the [COLOR_BUILDING_TEXT]Taoist Gong[COLOR_REVERT] Building.</English>
```
Comment (dguenms) [inline Assets/XML/Text/Buildings.xml]: see above

File: Assets/XML/Text/Buildings.xml [hunk 13]
```diff
@@ -1060,7 +1104,7 @@
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_BUILDING_ZOROASTRIAN_TEMPLE_STRATEGY</Tag>
-		<English>[COLOR_BUILDING_TEXT]Fire Temples[COLOR_REVERT] ([COLOR_BUILDING_TEXT]Zoroastrian Temples[COLOR_REVERT]) increase a city's happiness and culture.</English>
+		<English>[COLOR_BUILDING_TEXT]Fire Temples[COLOR_REVERT] ([COLOR_BUILDING_TEXT]Zoroastrian Temples[COLOR_REVERT]) increase a city's happiness and culture. [COLOR_BUILDING_TEXT]Fire Temples[COLOR_REVERT] are required to build the [COLOR_BUILDING_TEXT]Zoroastrian Royal Fire[COLOR_REVERT] Building.</English>
```
Comment (dguenms) [inline Assets/XML/Text/Buildings.xml]: see above

File: Assets/XML/Text/Units.xml [hunk 14]
```diff
@@ -1340,7 +1340,7 @@
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_UNIT_ORTHODOX_MISSIONARY_STRATEGY</Tag>
-		<English>[COLOR_UNIT_TEXT]Orthodox Missionaries[COLOR_REVERT] can spread [COLOR_HIGHLIGHT_TEXT]Orthodoxy[COLOR_REVERT] in friendly cities (or rival cities with whom you have [COLOR_HIGHLIGHT_TEXT]Open Borders[COLOR_REVERT]).</English>
+		<English>[COLOR_UNIT_TEXT]Missionaries[COLOR_REVERT] ([COLOR_UNIT_TEXT]Orthodox Missionaries[COLOR_REVERT]) can spread [COLOR_HIGHLIGHT_TEXT]Orthodoxy[COLOR_REVERT] in Russia, Southern Europe, the Middle East, Central Asia and Alaska.</English>
```
Comment (dguenms) [inline Assets/XML/Text/Units.xml]: Please remove the region references, they are misleading (simply revert to the old text).

File: Assets/XML/Text/Units.xml [hunk 15]
```diff
@@ -1349,7 +1349,7 @@
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_UNIT_CATHOLIC_MISSIONARY_STRATEGY</Tag>
-		<English>[COLOR_UNIT_TEXT]Catholic Missionaries[COLOR_REVERT] can spread [COLOR_HIGHLIGHT_TEXT]Catholicism[COLOR_REVERT] in friendly cities (or rival cities with whom you have [COLOR_HIGHLIGHT_TEXT]Open Borders[COLOR_REVERT]).</English>
+		<English>[COLOR_UNIT_TEXT]Missionaries[COLOR_REVERT] ([COLOR_UNIT_TEXT]Catholic Missionaries[COLOR_REVERT]) can spread [COLOR_HIGHLIGHT_TEXT]Catholicism[COLOR_REVERT] in Europe, North America, South America, Oceania and Africa.</English>
```
Comment (dguenms) [inline Assets/XML/Text/Units.xml]: see above

File: Assets/XML/Text/Units.xml [hunk 16]
```diff
@@ -1358,7 +1358,7 @@
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_UNIT_PROTESTANT_MISSIONARY_STRATEGY</Tag>
-		<English>[COLOR_UNIT_TEXT]Protestant Missionaries[COLOR_REVERT] can spread [COLOR_HIGHLIGHT_TEXT]Protestantism[COLOR_REVERT] in friendly cities (or rival cities with whom you have [COLOR_HIGHLIGHT_TEXT]Open Borders[COLOR_REVERT]).</English>
+		<English>[COLOR_UNIT_TEXT]Missionaries[COLOR_REVERT] ([COLOR_UNIT_TEXT]Protestant Missionaries[COLOR_REVERT]) can spread [COLOR_HIGHLIGHT_TEXT]Protestantism[COLOR_REVERT] in Europe, North America, Oceania and South Africa.</English>
```
Comment (dguenms) [inline Assets/XML/Text/Units.xml]: see above

File: Assets/XML/Text/Units.xml [hunk 17]
```diff
@@ -1367,7 +1367,7 @@
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_UNIT_ISLAMIC_MISSIONARY_STRATEGY</Tag>
-		<English>[COLOR_UNIT_TEXT]Imams[COLOR_REVERT] ([COLOR_UNIT_TEXT]Islamic Missionaries[COLOR_REVERT]) can spread [COLOR_HIGHLIGHT_TEXT]Islam[COLOR_REVERT] in friendly cities (or rival cities with whom you have [COLOR_HIGHLIGHT_TEXT]Open Borders[COLOR_REVERT]).</English>
+		<English>[COLOR_UNIT_TEXT]Imams[COLOR_REVERT] ([COLOR_UNIT_TEXT]Islamic Missionaries[COLOR_REVERT]) can spread [COLOR_HIGHLIGHT_TEXT]Islam[COLOR_REVERT] in the Middle East, Central Asia, India, South-East Asia, Russia, Ethiopia, South Europe and West Africa.</English>
```
Comment (dguenms) [inline Assets/XML/Text/Units.xml]: see above

File: Assets/XML/Text/Units.xml [hunk 18]
```diff
@@ -1403,7 +1403,7 @@
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_UNIT_TAOIST_MISSIONARY_STRATEGY</Tag>
-		<English>[COLOR_UNIT_TEXT]Daoshis[COLOR_REVERT] ([COLOR_UNIT_TEXT]Taoist Missionaries[COLOR_REVERT]) can spread [COLOR_HIGHLIGHT_TEXT]Taoism[COLOR_REVERT] in friendly cities (or rival cities with whom you have [COLOR_HIGHLIGHT_TEXT]Open Borders[COLOR_REVERT]).</English>
+		<English>[COLOR_UNIT_TEXT]Daoshis[COLOR_REVERT] ([COLOR_UNIT_TEXT]Taoist Missionaries[COLOR_REVERT]) can spread [COLOR_HIGHLIGHT_TEXT]Taoism[COLOR_REVERT] in China, Tibet and Central Asia.</English>
```
Comment (dguenms) [inline Assets/XML/Text/Units.xml]: see above

File: Assets/XML/Text/Units.xml [hunk 19]
```diff
@@ -1412,6 +1412,6 @@
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_UNIT_ZOROASTRIAN_MISSIONARY_STRATEGY</Tag>
-		<English>[COLOR_UNIT_TEXT]Magus[COLOR_REVERT] ([COLOR_UNIT_TEXT]Zoroastrian Missionaries[COLOR_REVERT]) can spread [COLOR_HIGHLIGHT_TEXT]Zoroastrianism[COLOR_REVERT] in friendly cities (or rival cities with whom you have [COLOR_HIGHLIGHT_TEXT]Open Borders[COLOR_REVERT]).</English>
+		<English>[COLOR_UNIT_TEXT]Magus[COLOR_REVERT] ([COLOR_UNIT_TEXT]Zoroastrian Missionaries[COLOR_REVERT]) can spread [COLOR_HIGHLIGHT_TEXT]Zoroastrianism[COLOR_REVERT] in Persia and India.</English>
```
Comment (dguenms) [inline Assets/XML/Text/Units.xml]: see above

**Comment by SgtBears** [comment]
Fixed. I also fixed the Soundtrack and VD update.

**Comment by SgtBears** [comment]
I tested the soundtrack and the sound works correctly!

**Comment by dguenms** [comment]
Awesome. I'll go ahead and merge.

**Pull request merged**